README.md

```diff
@@ -1,14 +1,58 @@
-# Prometheus Integration Pack
+# prometheus integration pack v0.2.2
```

Comment:
```suggestion
# Prometheus Integration Pack
```
The pack name is really `Prometheus Integration Pack`, and not the `prometheus integration pack v0.2.2`.
This is counter-intuitive. The versioning should be part of the `pack.yml`, and not the title.

Reply:
I think it's a useful bit of information to know what version of the project the documentation refers to.  The version can be moved to another line if you don't want it as part of the title.  As for the capitalisation, that comes from the pack.yaml so this capitalisation can be fixed in there if this is an issue.

Reply:
There looks like already established practice of how Readme and Titles are organized in StackStorm Exchange. Please follow that format.

The version is already part of the `pack.yml`, git tags and github releases.